### PR TITLE
UID2-3308 SDK doc changes: domain name parameter can be appname or null

### PR DESCRIPTION
**Pull request by jon8787**

### CODE REVIEW

File: docs/ref-info/glossary-uid.md [hunk 1]
```diff
@@ -112,6 +113,9 @@ import Link from '@docusaurus/Link';
 <dt><MdxJumpAnchor id="gl-api-secret">API secret</MdxJumpAnchor></dt>
 <dd>See <a href="#gl-client-secret">client secret</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-app-name">App name</MdxJumpAnchor></dt>
+<dd>The Android application ID,  the iOS app store ID, or the iOS bundle identifier.</dd>
```
Comment (sunnywu) [inline docs/ref-info/glossary-uid.md:117]: is that extra spaces before "the iOS app store ID" ?

**Review by sunnywu — APPROVED** [review]

File: docs/ref-info/glossary-uid.md (hunk 1 above)
Comment (jon8787) [reply docs/ref-info/glossary-uid.md]: thanks, fixed

File: docs/sdks/uid2-sdk-ref-csharp-dotnet.md [hunk 2]
```diff
@@ -145,10 +145,14 @@ var client = new BidstreamClient(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY);
 client.Refresh();
 ```
 
-3. Decrypt a token into a raw UID2. Pass the token, and the domain name of the site where the bid originated from. The domain name must be all lower case, without spaces and without subdomain. For example, for `Subdomain.DOMAIN.com` , pass `domain.com` instead:
+3. Decrypt a token into a raw UID2. Pass the token, and:
```
[the remark this answers is not in the document]
Comment (jon8787) [reply docs/sdks/uid2-sdk-ref-csharp-dotnet.md]: thanks, done

File: docs/sdks/uid2-sdk-ref-csharp-dotnet.md [hunk 3]
```diff
@@ -145,10 +145,14 @@ var client = new BidstreamClient(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY);
 client.Refresh();
 ```
 
-3. Decrypt a token into a raw UID2. Pass the token, and the domain name of the site where the bid originated from. The domain name must be all lower case, without spaces and without subdomain. For example, for `Subdomain.DOMAIN.com` , pass `domain.com` instead:
+3. Decrypt a token into a raw UID2. Pass the token, and: 
+ * If the bid request originated from a publisher's website, pass the domain name. The domain name must be all lower case, without spaces and without subdomain. For example, for `Subdomain.DOMAIN.com`, pass `domain.com` instead.
```
[the remark this answers is not in the document]
Comment (jon8787) [reply docs/sdks/uid2-sdk-ref-csharp-dotnet.md:149]: eg it came from a server or a CTV

**Pull request merged**